app/config.go

```diff
@@ -0,0 +1,59 @@
+package app
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	dbm "github.com/tendermint/tm-db"
+
+	"github.com/certikfoundation/shentu/v2/app/params"
+)
+
+// DefaultConfig returns a default configuration suitable for nearly all
+// testing requirements.
+func DefaultConfig() network.Config {
+	encCfg := MakeEncodingConfig()
+
+	return network.Config{
+		Codec:             encCfg.Marshaler,
+		TxConfig:          encCfg.TxConfig,
+		LegacyAmino:       encCfg.Amino,
+		InterfaceRegistry: encCfg.InterfaceRegistry,
+		AccountRetriever:  authtypes.AccountRetriever{},
+		AppConstructor:    NewAppConstructor(encCfg),
+		GenesisState:      ModuleBasics.DefaultGenesis(encCfg.Marshaler),
+		TimeoutCommit:     1 * time.Second,
+		ChainID:           "shentu-code-test",
+		NumValidators:     1,
+		BondDenom:         "uctk",
+		MinGasPrices:      fmt.Sprintf("0.025%s", "uctk"),
```

Comment:
use common.MicroCTKDenom instead of "uctk" throughout the PR, in this file and in the test file.